sdk/cmake/config-amd64.cmake

```diff
@@ -44,5 +44,10 @@ set(USERMODE TRUE CACHE BOOL
 if(MSVC)
 set(_PREFAST_ FALSE CACHE BOOL
 "Whether to enable PREFAST while compiling.")
-    option(RUNTIME_CHECKS "Whether to enable runtime checks on MSVC" ON)
+    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+        option(RUNTIME_CHECKS "Whether to enable runtime checks on MSVC" ON)
+    else()
+        # RTC are incompatible with compiler optimizations.
+        set(RUNTIME_CHECKS FALSE CACHE BOOL "Whether to enable runtime checks on MSVC")
```

Comment:
Why does this not use `option` like the regular case?

Reply:
I felt like `set()` would be a bit more explicit about never changing the value.
If you prefer `option()`, that should work fine too.

Reply:
Sounds like a perfect use case for https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html#module:CMakeDependentOption

Reply:
Nice suggestion.
Done.